packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.js

```diff
@@ -0,0 +1,129 @@
+
+  /**
+   * @type {string}
+   */
+  @track type;
+  /**
+   * @type {Boolean}
+   */
+  @track hasError;
+  /**
+   * @type {string}
+   */
+  @track error
+
+  showMoreInfo = false;
+
+  labels = {
+    coveoOnlineHelpLink,
+    moreInformation,
+    checkForMore,
+    community,
+    contactCoveoSupportTeam,
+    goBack
+  }
+
+  connectedCallback() {
+    registerComponentForInit(this, this.engineId);
+  }
+
+  renderedCallback() {
+    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+  }
+
+  /**
+   * @param {import("coveo").SearchEngine} engine
+   */
+  @api
+  initialize(engine) {
+    this.queryError = CoveoHeadless.buildQueryError(engine);
+    this.unsubscribe = this.queryError.subscribe(() => this.updateState());
+  }
+
+  disconnectedCallback() {
+    if (this.unsubscribe) {
+      this.unsubscribe();
+    }
+  }
+
+  updateState() {
+    this.type = this.queryError.state.error?.type;
+    this.hasError = this.queryError.state.hasError;
+    this.error = this.queryError.state.error ? JSON.stringify(this.queryError.state.error, null, 2): "";
+  }
+
+  get errorTitle() {
+    switch (this.type) {
+      case disconnectedException:
+        return "Could not connect.";
+      case noEndpointsException:
+        return "The Coveo Organization has no registered endpoints.";
+      case invalidTokenException:
+        return "The Coveo Organization cannot be accessed.";
+      case organizationIsPausedException:
+        return "Your organization is resuming and will be available shortly.";
+      default:
+        return "Something went wrong.";
+    }
+  }
+
+  get description() {
+    switch (this.type) {
+      case disconnectedException:
+        return "Your query couldn't be sent. Verify your internet connection.";
+      case noEndpointsException:
+        return "You will need to add sources in your index, or wait for the created sources to finish indexing.";
+      case invalidTokenException:
+        return "The token is invalid.";
+      case organizationIsPausedException:
+        return "Your organization is resuming and will be available shortly.";
+      default:
+        return "If the problem persists contact the administrator.";
+    }
+  }
+
+  get link() {
+    switch (this.type) {
+      case noEndpointsException:
+        return "https://docs.coveo.com/";
+      case invalidTokenException:
+        return "https://docs.coveo.com/";
+      case organizationIsPausedException:
+        return "https://docs.coveo.com/l6af0467";
+      default:
+        return null;
+    }
+  }
+
+  handleShowMoreInfoClick() {
+    this.showMoreInfo = !this.showMoreInfo;
+  }
+
+  get checkForMoreLabel() {
+    return I18nUtils.format(this.labels.checkForMore, I18nUtils.getTextWithDecorator(this.labels.community, '<a href="https://connect.coveo.com/s/discussions">', '</a>'), I18nUtils.getTextWithDecorator(this.labels.contactCoveoSupportTeam,'<a href="https://connect.coveo.com/s/article/5382">', '</a>'));
```

Comment:
I end on this strange-looking page when following the link (.../discussions). The "Times New Roman" font and "Page not available" message look suspicious.

![connect-discussions](https://user-images.githubusercontent.com/10946843/127701066-b6d7a632-257f-4c6d-8e3c-c630d2bcc970.png)

Reply:
Maybe it's just https://connect.coveo.com/s/ ?
The times new roman font is a different issue 😂 I reported it to our website team, they were not installing the font Gibson, but were using it in the CSS without a fallback.